gnomad/assessment/validity_checks.py

```diff
@@ -982,3 +983,82 @@ def validate_release_t(
             t, info_metrics, non_info_metrics, n_sites, missingness_threshold
         )
     logger.info("VALIDITY CHECKS COMPLETE")
+
+
+def count_vep_annotated_variants_per_interval(
+    vep_ht: hl.Table, interval_ht: hl.Table
+) -> hl.Table:
+    """
+    Calculate the count of VEP annotated variants in `vep_ht` per interval defined by `interval_ht`.
+
+    :param vep_ht: VEP-annotated Table.
+    :param interval_ht: Interval Table.
+    :return: Interval Table with annotations for the counts of total variants and variants annotated as "protein-coding" in biotype.
+    """
+    logger.info(
+        "Selecting the vep.transcript_consequences field and joining with the"
+        " interval_ht..."
+    )
+
```

Comment:
I don't think you need this logger, or if you want a logger here, I would change it to:

```suggestion
    logger.info(
        "Counting the number of total variants and protein-coding variants in each interval..."
    )

```